sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py

```diff
@@ -102,7 +102,7 @@ def subscribe_port_update_event(self):
                 port_tbl.filter = d['FILTER'] if 'FILTER' in d else None
                 asic_context[port_tbl] = asic_id
                 sel.addSelectable(port_tbl)
-                self.logger.log_warning("subscribing to port_tbl {} - {} DB of namespace {} ".format(
+                self.logger.log_notice("subscribing to port_tbl {} - {} DB of namespace {} ".format(
```

Comment:
I would ideally like to keep it as INFO level messages rather than NOTICE level message unless it provides significant value by default.

Reply:
just this single log entry, or all of the ones I changed?

Reply:
for all of the ones

Reply:
done